### DISCUSSION TITLE
Ableton Live 12 Crash

### DISCUSSION THREAD

**Discussion by Faproger** [ZL-Audio/ZLEqualizer#71] [General]
Hi!

When i put ZL Equaliser on a MIDI or AUDIO track, ableton crash. 

Thx for your help.

**Comment by zsliu98** [comment]
Would you like to share the OS info (macOS or Win10/Win11) and the plugin version? Do 0.3.1 and 0.3.2 both crash the DAW or only 0.3.2?

**Reply by Faproger** [reply]
It's on Win 11 with only 0.3.2.

Thx

**Reply by zsliu98** [reply]
You may check Windows updates and install the [Latest Microsoft Visual C++ Redistributable](https://learn.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170). If the crash still happens after the updates, please kindly wait for the next(?) release. As I am not able to reproduce the issue locally, the problem might be on the JUCE side.

**Comment by Faproger** [comment]
Thank you very much, it works. 👍
